convert/convert.go

```diff
@@ -11,12 +11,21 @@ import (
 	"github.com/ollama/ollama/llm"
 )
 
-type Parameters struct {
+type ModelParameters struct {
```

Comment:
it's implicit (as it is elsewhere in the product) that we're dealing with models already
```suggestion
type Parameters struct {
```